docs/pages/management/admin/labels.mdx

```diff
@@ -249,6 +249,11 @@ you run `tsh ls`.
 
 </Details>
 
+<Details title="Hidden labels" opened={false}>
+
+Labels that should not be shown in the UI should be under the teleport.hidden namespace.
```

Comment:
Do we expect readers to know what the teleport.hidden namespace means? I would add more detail here.

Maybe something like this:

If you wish to use labels for RBAC purposes but don't want to see the labels
in the UI, you can put them in the hidden namespace by prefixing the label's
key with `teleport.hidden/`.

Reply:
Yes, we should explain in more detail what the implications are of using teleport.hidden/ label namespace.